sdk/python/touca/_transport.py

```diff
@@ -6,7 +6,7 @@
 from urllib3.poolmanager import PoolManager
 from urllib3.response import HTTPResponse
 
-__version__ = "1.8.2"
+__version__ = "1.8.3"
```

Comment:
Unrelated change: Bumping SDK version since I just released `v1.8.2`. For Python SDK, we only need to make this change in one file (this file) which is neat. The release process is very straightforward btw:
- Make sure Changelog is updated.
- Go to GitHub Actions "release-sdk-python" workflow and trigger a manual run of the workflow.
- git tag python-sdk-v1.x.x
- git push origin --tags